helper-cli/src/main/kotlin/commands/GenerateRuleViolationResolutionsCommand.kt

```diff
@@ -58,16 +60,19 @@ internal class GenerateRuleViolationResolutionsCommand : CliktCommand(
 
     private val severity by option(
         "--severity",
-        help = "Only consider violations of the given severity. Allowed values: ERROR|WARNING|HINT."
+        help = "Only consider violations of the given severities, specified as comma separated values. Allowed " +
+                "values: ERROR|WARNING|HINT."
```

Comment:
Nit: s/comma separated/comma-separated/

And as you talk about comma-separated values, how about also doing s/ERROR|WARNING|HINT/ERROR,WARNING,HINT/?

Reply:
Done.